backend/x11/meson.build

```diff
@@ -10,16 +10,16 @@ x11_required = [
 ]
 
 msg = []
-if get_option('x11-backend').enabled()
-	msg += 'Install "@0@" or pass "-Dx11-backend=disabled" to disable it.'
-endif
-if not get_option('x11-backend').disabled()
-	msg += 'Required for X11 backend support.'
+
+if 'x11'.contains(backends)
```

Comment:
This looks like it will evaluate to true if `backends` is equal to or a substring of "x11". I imagine you wanted it to be true if one of the elements in `backends` was equal to "x11". Am I missing some magic here?